app/models/institution_builder.rb

```diff
@@ -383,7 +383,7 @@ def self.add_hcm(version_number)
     str = <<-SQL
       UPDATE institutions SET
         caution_flag = TRUE,
-        caution_flag_reason = concat_ws(',', caution_flag_reason, hcm_list.reasons)
+        caution_flag_reason = concat_ws(', ', caution_flag_reason, hcm_list.reasons)
```

Comment:
Would be nice to add a spec covering this concatenation to make sure it looks right.